Extensions/Guardrail+Settings.swift

```diff
@@ -143,4 +154,10 @@ public extension Guardrail where Value == HKQuantity {
             unit: .internationalUnit()
         )
     }
+    
+    static func selectableBolusVolumes(supportedBolusVolumes: [Double]) -> [Double] {
+        return supportedBolusVolumes.filter {
+            Guardrail.maximumBolus(supportedBolusVolumes: supportedBolusVolumes).absoluteBounds.contains(HKQuantity(unit: .internationalUnit(), doubleValue: $0))
```

Comment:
[nit] Pull out the result of `Guardrail.maximumBolus(supportedBolusVolumes: supportedBolusVolumes).absoluteBounds` into local value so it isn't recalculated for every item in `supportedBolusVolumes`?

Reply:
[nit] It is a bit confusing why "selectable bolus volumes" would only be in the range of "maximum bolus values". I understand it after thinking about it (i.e. it isn't related to the maximum bolus itself, but rather the range of valid maximum bolus values).

Reply:
There's likely a bit of history here, where "selectable values" preceded "guardrails", which is why there's both.  The assumption is that you can only select values that fall within the absolute bounds of guardrails, which to me seems like a `Guardrails` type of responsibility.  Could this be cleaned up?  Definitely.  But, again, I'm doing the minimal changes to get us to 510k.